src/dispatch.rs

```diff
@@ -466,6 +468,7 @@ pub(crate) fn gen_dispatch_str(cmd: DispatchType, dispatch: bool) -> crate::Resu
         Global(name) => format!("global{sep}{name}"),
         KillActiveWindow => "killactive".to_string(),
         CloseWindow(win) => format!("closewindow{sep}{win}"),
+        ActiveWorkspace => "activeworkspace".to_string(),
```

Comment:
Its not a dispatcher, instead this PR should replace the helper implementation found at https://github.com/hyprland-community/hyprland-rs/blob/5c9d0125f9a96f6f35df7cbd049ac59e75716ef3/src/data/regular.rs#L173

Reply:
Ok ! I give to you so 😄